packages/remark-math/package.json

```diff
@@ -33,7 +33,10 @@
     "util.js"
   ],
   "main": "index.js",
+  "types": "types/index.d.ts",
   "dependencies": {},
-  "devDependencies": {},
+  "peerDependencies": {
+    "unified": "^9.0.0"
```

Comment:
remark/unified generally doesn't use peerDependencies, they can cause trouble.
Generally `dependencies` or `devDependencies` are used.

Reply:
ok. I'll move it to dependencies.

Reply:
Sorry, I want to reopen this!

While I think including a package for types is fine (e.g., types/hast, vfile). I don’t think we should do it for unified. I think it should be removed from dependencies, and a note added in the readme that they should be installed (although: plugins don’t make sense w/o unified, so they probably always are installed).